src/main/java/org/folio/dew/controller/UploadController.java

```diff
@@ -35,35 +35,36 @@
 import lombok.extern.log4j.Log4j2;
 
 import static java.lang.String.format;
-import static java.time.format.DateTimeFormatter.ofPattern;
 import static java.util.Optional.ofNullable;
 import static java.util.stream.Collectors.joining;
 import static org.folio.dew.domain.dto.ExportType.BULK_EDIT_IDENTIFIERS;
 import static org.folio.dew.domain.dto.ExportType.BULK_EDIT_QUERY;
+import static org.folio.dew.domain.dto.ExportType.BULK_EDIT_UPDATE;
 import static org.folio.dew.domain.dto.JobParameterNames.TEMP_OUTPUT_FILE_PATH;
 import static org.folio.dew.utils.Constants.EXPORT_TYPE;
 import static org.folio.dew.utils.Constants.FILE_NAME;
+import static org.folio.dew.utils.Constants.ROLLBACK_FILE;
+import static org.folio.dew.utils.Constants.TMP_DIR_PROPERTY;
+import static org.folio.dew.utils.Constants.PATH_SEPARATOR;
 
 @RestController
 @RequestMapping("/bulk-edit")
 @Log4j2
 @RequiredArgsConstructor
 public class UploadController implements JobIdApi {
```

Comment:
UploadController was responsible for uploading a file but now it contains preview logic and added rollback logic. 
The controller should be renamed to BulkEditController or a separate controller should be created.

Reply:
Done